### ISSUE TITLE
03_claasification-spam_classifier

### ISSUE THREAD

**Issue by d5423197** [ageron/handson-ml2#285]
for word_count in X:
            for word, count in word_count.items():
                total_count[word] += min(count, 10)

Hello there,

I have a question regarding "min(count, 10)".

Why we only output10 if the count is greater than 10?

Thanks,

**Comment by Praful932** [comment]
@d5423197 , As per my knowledge, it is kept as a threshold. If not there would be a sentence which would highly skew the features.

**Comment by d5423197** [comment]
> @d5423197 , As per my knowledge, it is kept as a threshold. If not there would be a sentence which would highly skew the features.

I do not think that way. I think we should just keep the count and add them up. For example, if one word is 1+11 which will be 11 and another word is 1+19 which also is 11 in that case.
